hack/generate/samples/internal/go/memcached-with-webhooks/memcached_with_webhooks.go

```diff
@@ -87,18 +87,19 @@ func (mh *Memcached) Run() {
 		"--group", mh.ctx.Group,
 		"--version", mh.ctx.Version,
```

Comment:
@NikhilSharmaWe we need to update the mock for the e2e tests here: https://github.com/operator-framework/operator-sdk/blob/master/testdata/go/v3/memcached-operator/test/e2e/e2e_test.go#L163-L177

Because with the plugin the label to create the deployment for the CR is not the same, we need to have this check https://github.com/kubernetes-sigs/kubebuilder/blob/master/test/e2e/deployimage/plugin_cluster_test.go#L253-L265 instead.